cmd/server.go

```diff
@@ -52,6 +53,9 @@ const (
 	// JamulusBridgeServiceName is the name of the systemd service for the Jamulus -> JackTrip  bridge
 	JamulusBridgeServiceName = "jamulus-bridge.service"
 
+	// PathToServerCredentials is the path to jacktrip-agent server credentials file
+	PathToServerCredentials = "/etc/default/jacktrip-agent-server"
```

Comment:
i wonder if we should name this something more descriptive of what it contains, like `jacktrip-token` or `jacktrip-agent-token` or `jacktrip-studio-token` or ... ?